src/game/protos/object.c

```diff
@@ -411,11 +411,6 @@ void object_render(object *obj) {
         remap_rounds = 0;
         remap_offset = 5;
         options |= SPRITE_REMAP | SPRITE_DARK_TINT;
-    } else if(object_has_effect(obj, EFFECT_POSITIONAL_LIGHTING)) {
-        int rx = x + (w >> 1);
-        remap_rounds = 0;
```

Comment:
There were some signs in the original game engine that the stadium lighting does in fact use the remaps, which is why we used it here. We didn't have the time to figure out why the effect was too dark. One guess was that we would need to run the remapping multiple times.

Reply:
^ Agreed with the above.